packages/invoices/src/OutboundShipment/DetailView/tabs/GeneralTab.tsx

```diff
@@ -59,13 +59,13 @@ export const GeneralTabComponent: FC<
   const t = useTranslation('distribution');
   const { isGrouped, toggleIsGrouped } = useIsGrouped('outboundShipment');
 
-  const activeRows = useMemo(() => {
-    const x = data
-      .filter(({ isDeleted }) => !isDeleted)
-      .slice(pagination.offset, pagination.offset + pagination.first);
-
-    return x;
-  }, [data]);
+  const activeRows = useMemo(
```

Comment:
one that will be replaced anyway I expect.
There's a funny bug with the outbound - if you view a detail page, go back to the list view and return to the same detail, the number of rows is added to. Thought that this will probably be fixed with the refactor 🤞

Reply:
Ah yes most likely